### PR TITLE
new module that allows for adding DynamicAddressGroup tags to the firewall

### PR DESCRIPTION
**Pull request by ivanbojer**
##### SUMMARY
Create the ip address to tag associations. Tags will in turn be used to create DAG's

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME
panos_dag_tags.py

##### ANSIBLE VERSION
```
ansible 2.4.2.0
  config file = None
  configured module search path = [u'/Users/ibojer/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/ibojer/.virtualenvs/ansible/lib/python2.7/site-packages/ansible
  executable location = /Users/ibojer/.virtualenvs/ansible/bin/ansible
  python version = 2.7.13 (default, Feb  6 2017, 11:22:10) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
```

### CODE REVIEW

**Comment by ivanbojer** [comment]
ready_for_review

**Comment by vinayvenkat** [comment]
shipit

File: lib/ansible/modules/network/panos/panos_dag_tags.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    if exc:
+        return False, exc
+
+    return True, exc
+
+
+def main():
+    argument_spec = dict(
+        ip_address=dict(required=True),
+        password=dict(required=True),
```
Comment (gundalow) [inline lib/ansible/modules/network/panos/panos_dag_tags.py]: `, no_log=True`

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by ivanbojer** [comment]
ready_for_review

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Rebase then we can merge.
In the future ignore the CHANGELOG.md, update that in a separate PR as it's often a source of merge conflicts

**Comment by ivanbojer** [comment]
ready_for_review

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged, will need tests adding ASAP. We can guide you with this if needed.

**Pull request merged**